shared/src/main/scala/squants/package.scala

```diff
@@ -75,6 +75,33 @@ package object squants {
   type Price[A <: Quantity[A]] = squants.market.Price[A]
 
   /**
+    * Helper function to achieve uniform Double formatting over JVM and JS platforms.
+    * Simple Double.toString will format 1.0 as "1.0" on JVM and as "1" on JS
+    * @param d Double number to be formatted
+    * @return
+    */
+  private[squants] def crossFormat(d: Double): String = {
```

Comment:
This version of `crossFormat` isn't the same as before. Did something change in scala native that required us to change?